### PR TITLE
Bug 1209210 - Sanitize template string for the metadata.

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=382731ecb62892d1e7cac8b2eb866ff6a1e7fb9b

**Comment by mozfreddyb** [comment]
The current approach will *still* alert our linter, since you are assigning a variable to innerHTML.

I suggest you do the following:
```js
var html = [];
this.albums.forEach((album) => {
    var template = Sanitizer.createSafeHTML`<a class.....`
    html.push(template);
});
this.tiles.innerHTML = Sanitizer.unwrapSafeHTML(html);
```

This will not make eslint complain.

**Comment by mozfreddyb** [comment]
Actually, this needs https://bugzilla.mozilla.org/show_bug.cgi?id=1209416

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c5ca8996b7ecc7eaffb9f94626c905c3363ce700

**Comment by hfiguiere** [comment]
This address the feedback. But I had to cherry-pick the dependent bug.

File: shared/js/sanitizer.js [hunk 1]
```diff
@@ -74,8 +74,11 @@
      * Unwrap safe HTML created by createSafeHTML or a custom replacement that
      * underwent security review.
      */
-    unwrapSafeHTML: function (htmlObject) {
-      return htmlObject.__html;
+    unwrapSafeHTML: function (...htmlObjects) {
+      var markupList= htmlObjects.map(function(obj) {
```
Comment (justindarc) [inline shared/js/sanitizer.js]: nit: space before `=`

File: shared/js/sanitizer.js [hunk 2]
```diff
@@ -74,8 +74,11 @@
...
+        return  obj.__html;
```
Comment (justindarc) [inline shared/js/sanitizer.js]: nit: one space after `return`

File: shared/js/sanitizer.js (hunk 1 above)
Comment (hfiguiere) [inline shared/js/sanitizer.js]: this is not part of my patch. :-/ I'm blocked on it

Comment (mozfreddyb) [inline shared/js/sanitizer.js]: Sorry for the delay. Bug 1209416 is ready to be checked in now, just waiting for a sherrif to land it.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4fee02d9fd7aa8fd5eb4b59e95e264b4a2b6132a

**Pull request merged**